composer.json

```diff
@@ -1 +1,37 @@
-{}
\ No newline at end of file
+{
+    "name": "altapay/altapay-for-woocommerce",
+    "license": "MIT",
+    "description": "AltaPay: Payments less complicated",
+    "keywords": [
+        "payment",
+        "altapay"
+    ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "config": {
+        "preferred-install": "dist",
+        "sort-packages": true
+    },
+    "archive": {
+        "exclude": [
+            "phpstan.neon.dist",
+            "phpunit.xml.dist"
+        ]
+    },
+    "require": {
+        "php": "^5.6 || ^7.0",
+        "guzzlehttp/psr7": "^1.4",
+        "altapay/api-php":"^3.0",
+        "psr/log": "^1.1"
```

Comment:
I don't see where guzzlehttp/psr7 and psr/log are being used, are thease also just dependencies of dependencies?